packages/synapse-react-client/src/components/HasAccess/HasAccessV2.tsx

```diff
@@ -137,10 +187,18 @@ export function HasAccessV2(props: HasAccessProps) {
     AccessRequirement[]
   >([])
 
-  const { entityId, showButtonText = true } = props
-
+  const {
+    entityId,
+    showButtonText = true,
+    hideAccessIconForExternalFileHandle,
```

Comment:
Set the default value here so you don't have to do `??` or anything like that
```suggestion
    hideAccessIconForExternalFileHandle = false,
```